Source/WebCore/dom/Document.h

```diff
@@ -80,6 +80,14 @@
 #include <wtf/ThreadingPrimitives.h>
 #endif
 
+#if USE(APPLE_INTERNAL_SDK)
+#include <WebKitAdditions/DocumentAdditions.h>
+#else
+#define DOCUMENT_ADDITIONS_1
+#define DOCUMENT_ADDITIONS_2
+#define DOCUMENT_ADDITIONS_3
```

Comment:
The only change I'd suggest is instead of using "`_1`", "`_2`", "`_3`", use somewhat meaningful names:

```
DOCUMENT_ADDITIONS_INSTANCE_VARIABLES
DOCUMENT_ADDITIONS_METHOD_IMPLEMENTATIONS
DOCUMENT_ADDITIONS_CONSTRUCTOR_INITIALIZATIONS
```